compiler/plugins/target/ROCM/ROCMTargetUtils.cpp

```diff
@@ -180,6 +208,24 @@ void linkROCDLIfNecessary(llvm::Module *module, std::string targetChip,
   };
 }
 
+// Links optimized Ukernel bitcodes into the given module if the module needs
+// it.
+void linkUkernelBCIfNecessary(llvm::Module *module, Location loc,
```

Comment:
Nit: It seems like you only call this if the ukernels are needed anyway. Can just rename to `linkUKernelBCFiles`.

Reply:
Done!